commercetools-models/src/test/java/io/sphere/sdk/carts/CartFixtures.java

```diff
@@ -155,6 +158,59 @@ public static void withLineItemAndCustomLineItemFilledCart(final BlockingSphereC
         });
     }
 
+    public static void withDiscountedLineItem(final BlockingSphereClient client, final RelativeCartDiscountValue relativeCartDiscountValue, final Consumer<Cart> consumer) {
```

Comment:
maybe that is still not clear enough if it is a cart discount or a product discount, maybe `withCartHavingCartDiscountedLineItem`